functions/helpers/firebaseHelpers.js

```diff
@@ -178,16 +179,16 @@ const createDonor = function(params) {
   });
 };
 
-const getDonorID = (email) => {
-  return admin.auth().getUserByEmail(email)
```

Comment:
I refactored a few times when debugging and used `await` instead of `.then` to step through it more clearly.  I can change back to stay inline with the rest of the file if desired